src/main/kotlin/lotto/domain/Ticket.kt

```diff
@@ -1,12 +1,24 @@
 package lotto.domain
 
+import lotto.controller.WinningNumbers
+
 class Ticket(val lottoNumber: Set<LottoNumber>) {
     constructor() : this(generateLottoNumber())
```

Comment:
It's a nice approach to distinguish between generating a ticket randomly and manually.
However, I think the current constructor doesn't clearly convey that intent.
How about using a factory function instead?

Here’s a reference you might find helpful:
- [[Kotlin Docs] Coding convention - Factory functions](https://kotlinlang.org/docs/coding-conventions.html#factory-functions)